src/autocomplete.css

```diff
@@ -29,6 +29,14 @@
   outline: 3px solid #ffbf47;
 }
 
+.autocomplete__input.autocomplete__input--show-all-values {
```

Comment:
I need it for the padding :) otherwise it's overridden by `.autocomplete__input` later on